bythepixel/kyrio-go2broadband
# Affiliate Page

Closes #11 

## How should the reviewer test this?
- Navigate to http://dev.bythepixel.com/kyrio/ and http://dev.bythepixel.com/kyrio/affiliates/
- Ensure development matches the design in #11

## QA Checklist
